cypress/integration/userCanSeeTheMenuList.feature.js

```diff
@@ -0,0 +1,22 @@
+describe('User can see menu', () => {
+  beforeEach(() => {
+    cy.visit("/")
+    cy.get("[data-cy=menu]").click()
+  });
+
+  it('is expected to display the main menu', () => {
+    cy.get("[data-cy=main-menu]").should("contain", "Main Menu")
+  });
+
+  it('is expected to show the first item on the menu', () => {
+    cy.get("[data-cy=item-1").within(() =>{
+      cy.get("[data-cy=id]").should("contain", 1)
+      cy.get("[data-cy=dish]").should("contain","Kangaroo Steak")
```

Comment:
This test is not passing for you, right? We can't merge PR's with failing tests. Either remove the rest or implement the functionality

Reply:
Test file removed for now